src/app/components/ac-button/ac-button.component.scss

```
@@ -0,0 +1,43 @@
+@import '../../../scss/core';
+
+:host {
+  font-family: $primary-font;
+  button {
+    align-items: center;
+    border-radius: 18px;
```

não devemos usar o EM?
```suggestion
    border-radius: 2em;
```